build-package.sh

```diff
@@ -169,16 +169,22 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 # Generate jazzy docs (macOS) where the 'documentation' tag exists on the issue label
-if [ "$(uname)" == "darwin"* ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
-    if  [ -z "${GITHUB_USERNAME}" ] && [ -z "${GITHUB_PASSWORD}" ]; then
-
+if [ "$(uname)" == "Darwin" ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+    if  [ -n "${GITHUB_USERNAME}" -a -n "${GITHUB_PASSWORD}" ]; then
```

Comment:
Fixes inverted logic for whether this env vars are set (note - they are not - which is the next issue that needs to be addressed)